src/app/shared/ag-grid-renderers/vacation-renderer/vacation-renderer.component.html

```diff
@@ -0,0 +1,3 @@
+<div>
+  {{value}} {{value===1 ? ' day' : ' days'}}
```

Comment:
We should not create a renderer just to show the "day" suffix, please look into other alternatives and discuss with me